src/utils/registry.ts

```diff
@@ -0,0 +1,22 @@
+import { chainMetadata } from "@hyperlane-xyz/registry";
+import type { ChainMetadata, ChainName } from "@hyperlane-xyz/sdk";
+import { useMemo } from "react";
+
+const ABACUS_WORKS_DEPLOYER_NAME = "abacus works";
+
+// Chains that Abacus Works has deployed (formerly known as 'core' chains)
+export function getAbacusWorksChains(isTestnet = false): ChainMetadata[] {
+  return Object.values(chainMetadata).filter(
+    (metadata: ChainMetadata) =>
+      metadata.deployer?.name.trim().toLowerCase() ===
+        ABACUS_WORKS_DEPLOYER_NAME && !!metadata.isTestnet === isTestnet
+  );
```

Comment:
would also add a check that there is an address for the `mailbox` - sometimes we might add the metadata before doing the deploy and I think we don't want to show those as an AW chain til we merge the contract addresses